content/workshops/openshift_4_101/lab2-byocontainer.md

```diff
@@ -57,6 +57,11 @@ ore of the commands below:
 
 {{% panel "Web Console Steps" %}}
 
+<blockquote>
+Switch to "Developer" mode, by clicking on the menu, in the top left corner, where it says "Administrator", and pick "Developer"
```

Comment:
To switch to Developer mode, select "Developer" from the drop-down menu (top left of the screen).